src/inference/src/ie_core.cpp

```diff
@@ -1065,6 +1064,26 @@ class CoreImpl : public ie::ICore, public std::enable_shared_from_this<ie::ICore
         }
     }
 
+    /**
+     * @brief Get device config it is passed as pair of device_name and `AnyMap`
+     * @param configs All set of configs
+     * @note  `device_name` is not allowed in form of MULTI:CPU, HETERO:GPU,CPU, AUTO:CPU
+     *        just simple forms like CPU, GPU, MULTI, GPU.0, etc
+     */
+    void ExtractAndSetDeviceConfig(const ov::AnyMap& configs) {
+        for (auto&& config : configs) {
+            auto parsed = parseDeviceNameIntoConfig(config.first);
+            auto devices = GetListOfDevicesInRegistry();
+            auto config_is_device_name_in_regestry =
+                std::any_of(devices.begin(), devices.end(), [&](const std::string& device) {
+                    return device == parsed._deviceName;
+                });
+            if (config_is_device_name_in_regestry) {
+                SetConfigForPlugins(any_copy(config.second.as<ov::AnyMap>()), config.first);
```

Comment:
I suppose here you set config globally, while `ExtractAndSetDeviceConfig` is called in `LoadNetwork` which must NOT set global configs

Reply:
There is no way to implement it till we have only strings in the internal API. What about making a note that `ov::device::property` will change plugin state?

Reply:
let's just remove `ov::device::property` until plugins are ready for it

Reply:
fixed